### PR TITLE
Added CODE_OF_CONDUCT and CONTRIBUTING

### PR DESCRIPTION
**Pull request by xanjohns**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+following command to add signoffs to all commits between you and the upstream
+master:
+
+    git rebase --signoff upstream/master
+
+
+----
+
+This file is generated from [README.md](README.md), please edit that file then
+run the `./.github/update-contributing.py`.
```
Comment (ryancj14) [inline CONTRIBUTING.md]: What is meant by these lines? Also, which repo are you pulling from mainly for this file?

**Comment by ryancj14** [comment]
Also, it doesn't look like your commit is signed off. Remember you can use git cm which should include the -s (check your .gitconfig to make sure thats the right one).

File: CONTRIBUTING.md (hunk 1 above)
Comment (xanjohns) [reply CONTRIBUTING.md]: This example is used in a few places I believe, but I took it mainly from the project Xray repo. In that repo they just put the Contributing info at the end of the readme, and then run the python script to publish that specific section under CONTRIBUTING.md. Honestly, it seems like more of a hassle to me, but if the Contributing info needs to be included in the readme, this is probably the easiest way of doing it. 

I'll look around at some of the other repos to see if they all have contributing info in the readme as well. If not, we can just delete these lines.

Comment (ryancj14) [reply CONTRIBUTING.md]: sounds good

**Pull request merged**